.github/workflows/push.yml

```diff
@@ -0,0 +1,60 @@
+name: Push
+
+on: 
+  push:
+    branches:
+      - '*'
+    tags:
+      - '!v*'
+
+jobs:
+  linter:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/cache@v3
+      with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
I like to use `pipenv`, which we have the `Pipfile` and `Pipfile.lock` for.
If we're all using that it would be better to do this with `pipenv install`. Then we can even remove the entire `requirements.txt` imo.